examples/hello-world/hello-numpy-cross-val/README.md

```diff
@@ -27,3 +27,28 @@ $ ls /tmp/nvflare/simulate_job/
 app_server  app_site-1  app_site-2  log.txt
 
 ```
+
+# Run cross site validation using the previous trained results
+
+## Introduction
+
+The "hello-numpy-cross-val-only" and "hello-numpy-cross-val-only-list-models" jobs show how to run the NVFlare cross-site validation without the training workflow, making use of the previous run results. The first one uses the default single server model. The second enables a list of server models. You can provide / use your own previous trained models for the cross-validation.
+
+### Generate the previous run best global model and local best model
+
+Run the following command to generate the pre-trained models:
+
+```
+python pre_train_models.py 
```

Comment:
Thanks for adding this. Do we need to tell them what directory to run the script?

Reply:
don't have to. README also mentioned pointing "SERVER_MODEL_DIR" and "CLIENT_MODEL_DIR" to the proper folders. It could be any place.